README.md

```diff
@@ -54,6 +54,8 @@ JSON properties are written in the following format: `"key" [quantifier]: valueT
       "eventName": String,
       "startDate": unix_time,
```

Comment:
Auch, I duplicated the keys! Sorry, that was a mistake. Gonna fix that later at home. Thanks for the heads up!